src/frontend/src/pages/DesignReviewSummaryModal.tsx

```diff
@@ -0,0 +1,285 @@
+  const [documentState, setDocumentState] = useState<string>(() => designReview.docTemplateLink || '');
+  const [zoomState, setZoomState] = useState<string>(() => designReview.zoomLink || '');
+  const currentUser = useCurrentUser();
+  const toast = useToast();
+  const handleRemoveRequiredMember = (user: User) => {
+    if (currentUser.userId === designReview.userCreated.userId) {
+      setRequiredMembers(requiredMembers.filter((member) => member.userId !== user.userId));
+    } else {
+      toast.error('Only the creator of the Design Review can edit attendees');
+    }
+  };
+
+  const handleRemoveOptionalMember = (user: User) => {
+    if (currentUser.userId === designReview.userCreated.userId) {
+      setOptionalMembers(optionalMembers.filter((member) => member.userId !== user.userId));
+    } else {
+      toast.error('Only the creator of the Design Review can edit attendees');
+    }
+  };
+
+  const ModalButtons: React.FC = () => (
+    <Box
+      sx={{
+        position: 'absolute',
+        bottom: 10,
+        right: 10,
+        display: 'flex',
+        flexDirection: 'row'
+      }}
+    >
+      <NERButton
+        sx={{
+          textTransform: 'uppercase',
+          color: '#474849',
+          backgroundColor: '#d9d9d9',
+          ':hover': { backgroundColor: '#A4A4A4' },
+          fontSize: 10,
+          marginLeft: 1,
+          fontWeight: 'bold'
+        }}
+        disabled={designReview.status !== DesignReviewStatus.DONE || !checked}
+      >
+        Schedule Another DR
+      </NERButton>
+      <NERSuccessButton
+        sx={{
+          fontSize: 10,
+          marginLeft: 1,
+          fontWeight: 'bold'
+        }}
+        onClick={() => setShowStageGateModal(true)}
+        disabled={designReview.status !== DesignReviewStatus.DONE || !checked}
+      >
+        Stage Gate
+      </NERSuccessButton>
+      <NERFailButton
+        sx={{
+          fontSize: 10,
+          marginLeft: 1,
+          fontWeight: 'bold'
+        }}
+        onClick={() => setShowDelayModal(true)}
+        disabled={designReview.status !== DesignReviewStatus.DONE || !checked}
+      >
+        Request Delay to WP
+      </NERFailButton>
+    </Box>
+  );
+
+  const CheckBoxForm: React.FC = () => (
+    <FormControlLabel
+      label="Mark Design Review as Complete"
+      sx={{ marginBottom: 5 }}
+      control={
+        <Checkbox
+          checked={checked}
+          onChange={handleChange}
+          sx={{
+            color: 'inherit',
+            '&.Mui-checked': { color: 'inherit' }
+          }}
+        />
+      }
+    />
+  );
+
+  const MembersGrid: React.FC = () => (
+    <Box marginLeft="15px">
+      <Grid container direction="row" paddingY="20px">
+        <Grid item xs={12}>
+          <Grid container>
+            <Grid item sx={{ display: 'flex', alignItems: 'start', marginTop: '7px' }}>
+              <Typography>Required: </Typography>
+            </Grid>
+            <Grid item xs={10}>
```

Comment:
you can make this an item and a container, so you don't have to double up on the grids, same for the code below